prow/plank/reconciler.go

```diff
@@ -751,15 +727,13 @@ func predicates(additionalSelector string, callback func(bool)) (predicate.Predi
 }
 
 func podEventRequestMapper(prowJobNamespace string) handler.EventHandler {
-	return &handler.EnqueueRequestsFromMapFunc{
-		ToRequests: handler.ToRequestsFunc(func(mo handler.MapObject) []controllerruntime.Request {
-			return []controllerruntime.Request{{NamespacedName: ctrlruntimeclient.ObjectKey{
-				Namespace: prowJobNamespace,
-				Name:      mo.Meta.GetName(),
-			}},
-			}
-		}),
-	}
+	// TODO alvaroaleman: This shouldn't be taking a MapObject but an Object
```

Comment:
It looks like you already made that change here. (So, is the TODO still applicable?)

Reply:
good catch, removed it